src/Microsoft.OData.Edm/Csdl/CsdlXmlWriterSettings.cs

```diff
@@ -0,0 +1,19 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="CsdlXmlWriterSettings.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.Edm.Csdl
+{
+    /// <summary>
+    /// Provides settings to be used with <see cref="CsdlXmlWriter"/>.
+    /// </summary>
+    public class CsdlXmlWriterSettings
+    {
+        /// <summary>
+        /// Gets or sets the compatibility flags for the <see cref="CsdlXmlWriter"/>.

```

Comment:
I think we should also mention that these compatibility flags control whether we should retain compatibility with some legacy behaviour. Just my 2 cents.